docs/specification.md

```diff
@@ -459,6 +460,38 @@ type set_position_param = {
 type balance_nat = {x : nat ; y : nat}
 ```
 
+
+### **get_position_info**
```

Comment:
It would be nice to mention this entrypoint in the explanation in the Liquidity Mining section IMO